.github/workflows/automerge.yaml

```
@@ -0,0 +1,40 @@
+name: "Automerge"
+on:
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+      - synchronize
+      - opened
+      - edited
+      - ready_for_review
+      - reopened
+      - unlocked
+    branches:
+      - main
+      - 'release/*'
+    paths:
+      - 'services/**/*'
+  check_suite:
+    types:
+      - completed
+  workflow_dispatch: {}
+
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Enable automerge on automated chart bump PRs
+        id: automerge
+        uses: pascalgn/automerge-action@v0.15.3
+        if: startsWith(github.head_ref, 'chartbump/')
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          MERGE_LABELS: "!do-not-merge,!do-not-merge/testing"
```

❓ do we want to add `ready-for-review` as a required label for automerging? (automerge would be blocked if that label is missing even if all other checks, including number of approvals, is green)

---

I vote yes

---

we can always change it if we think the human action to add this label is slowing things down 😆 

```suggestion
          MERGE_LABELS: "ready-for-review,!do-not-merge,!do-not-merge/testing"
```